pandas/tests/groupby/test_grouping.py

```diff
@@ -149,6 +149,18 @@ def test_grouper_index_types(self):
             df.index = list(reversed(df.index.tolist()))
             df.groupby(list("abcde")).apply(lambda x: x)
 
+    def test_grouper_period_index(self):
+        # GH 32108
+        periods = 2
+        index = pd.period_range(start="2018-01", periods=periods, freq="M")
+        periodSerie = pd.Series(range(periods), index=index)
```

Comment:
naming this `periodSerie` doesn't conform to PEP8 - sorry for not bringing this up sooner, I thought flake8 would've caught it. seems there's an open issue to add extra plugins though (#20588)

Function and Variable Names
Function names should be lowercase, with words separated by underscores as necessary to improve readability.

> Variable names follow the same convention as function names.
> 
> mixedCase is allowed only in contexts where that's already the prevailing style (e.g. threading.py), to retain backwards compatibility.

https://www.python.org/dev/peps/pep-0008/

Reply:
Done, I hope it's all good now :)